datumaro/components/visualizer.py

```diff
@@ -1,18 +1,20 @@
 # Copyright (C) 2022 Intel Corporation
 #
 # SPDX-License-Identifier: MIT
-
 import math
 import warnings
 from collections import defaultdict
 from typing import Iterable, List, Optional, Tuple, Union
 
+import cv2
 import matplotlib.patches as patches
 import matplotlib.pyplot as plt
 import numpy as np
 from matplotlib.axes import Axes
 from matplotlib.figure import Figure
 from matplotlib.text import Text
+from mpl_toolkits.axes_grid1 import make_axes_locatable
```

Comment:
Is this installed in datumaro venv?

Reply:
It will be automatically installed if we install `matplotlib.pyplot`.

Reply:
great, thank you for clarification